docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,127 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Notation Used by SpECTRE {#spectre_notation_general}
+
+# Discontinuous Galerkin Notation {#dg_notation}
+
+- The logical coordinates in the element are
+  \f$\mathbf{\xi}=\{\xi,\eta,\zeta\}\f$.
+- The Jacobian matrix is defined as
+  \f{align*}{
+      \mathbf{J}=\frac{\partial \mathbf{x}}{\mathbf{\xi}}
+  \f}
+  where \f$\mathbf{x}\f$ are the coordinates mapped to by coordinate map.
+  The determinant of the Jacobian matrix is denoted by \f$J\f$.
+- We use \f$s\f$ and \f$t\f$ to be grid point indices. That is, \f$U_s\f$ is
```

Comment:
How about we use greek indices for these tensor product indices, since latin indices are already overloaded twice (spacetime and spatial) and we have no greek indices around so far? That's what we'll probably do for the elliptic DG paper.

Reply:
You should use whatever is in this document for SXS papers using SpECTRE. That's the point of this document.

Reply:
Edit: the elliptic dg paper by Trevor, Harald and myself that's in preparation right now

Reply:
Okay, since that's not using SpECTRE I don't think there's an issue with them doing whatever they want :)